packages/buefy-next/src/components/menu/MenuItem.vue

```diff
@@ -55,9 +55,9 @@ export default {
         }
     },
     // deprecated, to replace with default 'value' in the next breaking change
-    model: {
+    modelValue: {
         prop: 'active',
-        event: 'update:active'
```

Comment:
The [`model` section no longer works on Vue 3](https://v3-migration.vuejs.org/breaking-changes/v-model.html). I meant to replace the `active` prop with the `modelValue` prop; i.e.,

1. Remove the following lines: https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L57-L61

2. Replace the following prop with `modelValue`: https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L64

3. Tell Vue 3 the component may emit "update:modelValue" events instead of "update:active" events as you have already done in your PR: https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L89

4. Do not forget to emit "update:modelValue" events instead of "update:active" events (you can see this linter warning on the "Files changed" tab): https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L134

Reply:
This now makes more sense to me. I will make those changes soon

Reply:
> The [`model` option no longer works on Vue 3](https://v3-migration.vuejs.org/breaking-changes/v-model.html). There is no `modelValue` option neither. I meant to replace the `active` prop with the `modelValue` prop; i.e.,
> 
> 1. Remove the following lines: https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L57-L61
> 2. Replace the following prop with `modelValue`: https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L64
> 3. Tell Vue 3 the component may emit "update:modelValue" events instead of "update:active" events as you have already done in your PR: https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L89
> 4. Do not forget to emit "update:modelValue" events instead of "update:active" events (you can see this linter warning on the "Files changed" tab): https://github.com/ntohq/buefy-next/blob/bfac18dba589cdd2f42cdaae7168bdbb187e4297/packages/buefy-next/src/components/menu/MenuItem.vue#L134

I thought modelValue was an option based off of this source 

https://vuejs.org/guide/components/v-model

Reply:
`modelValue` is not a top-level "option" but a special "prop". You cannot do something like:
```js
export default {
  name: 'BMenuItem',
  // ...
  modelValue: {
    // ...
  },
  // ...
}
```
but you have do instead:
```js
export default {
  name: 'BMenuItem',
  // ...
  prop: {
    // ...
    modelValue: Boolean,
    // ...
  }
}
```